docs/develop-load-manager.md

```diff
@@ -207,3 +103,216 @@ then all machines are approximately overloaded. In the case in which a broker's
+
+:::note
+
+The pulsar-admin tool is not supported for enabling the extensible load manager.
+
+:::
+
+## Verification
+
+If you want to verify which load manager is used, follow the steps below.
+
+### Step 1: check loadManagerClassName
+
+You can use the `pulsar-admin` tool to examine the `loadManagerClassName` element.
+
+**Input**
+
+```bash
+bin/pulsar-admin brokers get-all-dynamic-config
+```
+
+**Output**
+
+```bash
+{
+"loadManagerClassName" : "org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl"
+}
+```
+
+If there is no `loadManagerClassName` element, then the value of [loadManagerClassName](https://github.com/apache/pulsar/blob/782e91fe327efe2c9c9107d6c679c2837d43935b/conf/broker.conf#L1309) in the `conf/broker.conf` file is used.
+
+### Step 2: verify load manager (optional) 
+
+To double-check which load manager is used, you can [check the ZooKeeper load report](#method-1-check-zookeeper-load-report) or [check monitor-brokers output](#method-2-check-monitor-brokers-output).  
+
+#### Method 1: check ZooKeeper load report
+
+Check the ZooKeeper load report. The load report in `/loadbalance/brokers/...`.
+
+```bash
+tbd
+```
+   
+Different load managers have different ZooKeeper load reports. 
+
+- [Simple load manager](./concepts-broker-load-balancing-types.md)
+
+  ```json
+  {
+    "systemResourceUsage": {
+      "bandwidthIn": {
+        "limit": 10240000.0,
+        "usage": 0.0
+      },
+      "bandwidthOut": {
+        "limit": 10240000.0,
+        "usage": 0.0
+      },
+      "cpu": {
+        "limit": 2400.0,
+        "usage": 0.0
+      },
+      "directMemory": {
+        "limit": 16384.0,
+        "usage": 1.0
+      },
+      "memory": {
+        "limit": 8192.0,
+        "usage": 3903.0
+      }
+    }
+  }
+  ```
+
+- [Modular load manager](./concepts-broker-load-balancing-types.md)
+
+   Modular load manager has different ZooKeeper load report from that of the simple load manager. For example, the `systemResourceUsage` sub-elements (`bandwidthIn`, `bandwidthOut`, etc.) are now all at the top level. 
+
+  ```json
+  {
+    "bandwidthIn": {
+      "limit": 10240000.0,
+      "usage": 4.256510416666667
+    },
+    "bandwidthOut": {
+      "limit": 10240000.0,
+      "usage": 5.287239583333333
+    },
+    "bundles": [],
+    "cpu": {
+      "limit": 2400.0,
+      "usage": 5.7353247655435915
+    },
+    "directMemory": {
+      "limit": 16384.0,
+      "usage": 1.0
+    }
+  }
+  ```
+
+- Extensible load manager
+
+  :::note
+
+  [Extensible load manager](./concepts-broker-load-balancing-types.md) does not have the ZooKeepr load report because its internal stats are stored in system topics rather than ZooKeeper.
+
+  :::
```

Comment:
@Demogorgon314 accurate?

Reply:
Yes

Reply:
Admins can run consumers on these topics and see the load report.

Reply:
@Demogorgon314 will test and try to give an extensible output example

Reply:
Input:
```bash
bin/pulsar-client consume non-persistent://pulsar/system/loadbalancer-broker-load-data --subscription-name test
```
Output:
```
2023-08-03T06:21:48,841+0000 [pulsar-client-io-1-3] INFO  org.apache.pulsar.client.impl.ConnectionPool - [[id: 0x69a65535, L:/127.0.0.1:59086 - R:localhost/127.0.0.1:6650]] Connected to server
2023-08-03T06:21:48,926+0000 [pulsar-client-io-1-3] WARN  org.apache.pulsar.client.impl.ConsumerImpl - [non-persistent://pulsar/system/loadbalancer-broker-load-data] Cannot create a [Durable] subscription for a NonPersistentTopic, will use [NonDurable] to subscribe. Subscription name: test
2023-08-03T06:21:49,189+0000 [pulsar-client-io-1-3] INFO  org.apache.pulsar.client.impl.ConsumerStatsRecorderImpl - Starting Pulsar consumer status recorder with config: {"topicNames":["non-persistent://pulsar/system/loadbalancer-broker-load-data"],"topicsPattern":null,"subscriptionName":"test","subscriptionType":"Exclusive","subscriptionProperties":null,"subscriptionMode":"NonDurable","receiverQueueSize":1000,"acknowledgementsGroupTimeMicros":100000,"maxAcknowledgmentGroupSize":1000,"negativeAckRedeliveryDelayMicros":60000000,"maxTotalReceiverQueueSizeAcrossPartitions":50000,"consumerName":null,"ackTimeoutMillis":0,"tickDurationMillis":1000,"priorityLevel":0,"maxPendingChunkedMessage":10,"autoAckOldestChunkedMessageOnQueueFull":false,"expireTimeOfIncompleteChunkedMessageMillis":60000,"cryptoFailureAction":"FAIL","properties":{},"readCompacted":false,"subscriptionInitialPosition":"Latest","patternAutoDiscoveryPeriod":60,"regexSubscriptionMode":"PersistentOnly","deadLetterPolicy":null,"retryEnable":false,"autoUpdatePartitions":true,"autoUpdatePartitionsIntervalSeconds":60,"replicateSubscriptionState":false,"resetIncludeHead":false,"batchIndexAckEnabled":false,"ackReceiptEnabled":false,"poolMessages":true,"startPaused":false,"autoScaledReceiverQueueSizeEnabled":false,"topicConfigurations":[],"maxPendingChuckedMessage":10}
2023-08-03T06:21:49,214+0000 [pulsar-client-io-1-3] INFO  org.apache.pulsar.client.impl.ConsumerStatsRecorderImpl - Pulsar client config: {"serviceUrl":"pulsar://localhost:6650/","authPluginClassName":null,"authParams":null,"authParamMap":null,"operationTimeoutMs":30000,"lookupTimeoutMs":30000,"statsIntervalSeconds":60,"numIoThreads":8,"numListenerThreads":8,"connectionsPerBroker":1,"connectionMaxIdleSeconds":180,"useTcpNoDelay":true,"useTls":false,"tlsKeyFilePath":"","tlsCertificateFilePath":"","tlsTrustCertsFilePath":"","tlsAllowInsecureConnection":false,"tlsHostnameVerificationEnable":false,"concurrentLookupRequest":5000,"maxLookupRequest":50000,"maxLookupRedirects":20,"maxNumberOfRejectedRequestPerConnection":50,"keepAliveIntervalSeconds":30,"connectionTimeoutMs":10000,"requestTimeoutMs":60000,"readTimeoutMs":60000,"autoCertRefreshSeconds":300,"initialBackoffIntervalNanos":100000000,"maxBackoffIntervalNanos":60000000000,"enableBusyWait":false,"listenerName":null,"useKeyStoreTls":false,"sslProvider":null,"tlsKeyStoreType":"JKS","tlsKeyStorePath":"","tlsKeyStorePassword":"*****","tlsTrustStoreType":"JKS","tlsTrustStorePath":"","tlsTrustStorePassword":"*****","tlsCiphers":[],"tlsProtocols":[],"memoryLimitBytes":0,"proxyServiceUrl":null,"proxyProtocol":null,"enableTransaction":false,"dnsLookupBindAddress":null,"dnsLookupBindPort":0,"socks5ProxyAddress":null,"socks5ProxyUsername":null,"socks5ProxyPassword":null,"description":null}
2023-08-03T06:21:49,236+0000 [pulsar-client-io-1-5] INFO  org.apache.pulsar.client.impl.ConnectionPool - [[id: 0xbeee21f2, L:/172.31.0.4:44850 - R:broker-1/172.31.0.4:6650]] Connected to server
2023-08-03T06:21:49,241+0000 [pulsar-client-io-1-5] INFO  org.apache.pulsar.client.impl.ConsumerImpl - [non-persistent://pulsar/system/loadbalancer-broker-load-data][test] Subscribing to topic on cnx [id: 0xbeee21f2, L:/172.31.0.4:44850 - R:broker-1/172.31.0.4:6650], consumerId 0
2023-08-03T06:21:49,273+0000 [pulsar-client-io-1-5] INFO  org.apache.pulsar.client.impl.ConsumerImpl - [non-persistent://pulsar/system/loadbalancer-broker-load-data][test] Subscribed to topic on broker-1/172.31.0.4:6650 -- consumer: 0
----- got message -----
key:[broker-1:8080], properties:[], content:{"cpu":{"usage":14.397985201479854,"limit":800.0},"memory":{"usage":300.0,"limit":2096.0},"directMemory":{"usage":40.0,"limit":256.0},"bandwidthIn":{"usage":0.7817884878178855,"limit":1.0E7},"bandwidthOut":{"usage":0.7213945272139455,"limit":1.0E7},"msgThroughputIn":3.000538680274058,"msgThroughputOut":3.0005621893825616,"msgRateIn":0.03333931866971176,"msgRateOut":0.033339579882028465,"bundleCount":3,"topics":3,"maxResourceUsage":0.15625,"weightedMaxEMA":0.15625,"msgThroughputEMA":38.88925615962549,"updatedAt":1691043751060,"reportedAt":1691043631073}
```